Cargo.toml

```
@@ -38,6 +38,8 @@ rev = "7dfa6d322b9df98b2d98afe0e14a97afe0187ac1"
     features = ["arduino-uno"]
   {%- when "SparkFun ProMicro" -%}
     features = ["sparkfun-promicro"]
+  {%- when "SparkFun ProMini 5v" -%}
+    features = ["sparkfun-promicro-5v"]
```

This needs to be fixed:

```suggestion
    features = ["sparkfun-promini-5v"]
```